src/hooks/useGa.ts

```diff
@@ -11,7 +11,12 @@ export const useGa = () => {
   // Enable/disable tracking
   useEffect(() => {
     if (IS_PRODUCTION && isAnalyticsEnabled) {
-      ReactGA.initialize(GOOGLE_ANALYTICS_TRACKING_ID)
+      ReactGA.initialize(GOOGLE_ANALYTICS_TRACKING_ID, {
+        gaOptions: {
+          cookieFlags: `domain=${GOOGLE_ANALYTICS_TRACKING_ID};SameSite=Strict;Secure`,
```

Comment:
```suggestion
          cookieFlags: `SameSite=Strict;Secure`,
```
Having this option results in no cookies being set for me. `cookieDomain` sets the domain for the cookie already so I think we can remove it here.